docs/adr/0001-simple-multi-tenancy-using-a-single-opencloud-instance.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1. Simple Multi-Tenancy using a single OpenCloud Instance"
+---
+
+* Status: proposed
+* Deciders: [@micbar @butonic @dragotin @rhafer]
+* Date: 2025-05-20
+
+Technical Story: https://github.com/opencloud-eu/opencloud/issues/877
+
+## Context and Problem Statement
+
+To reduce resource usage and cost service providers want a single OpenCloud
+instance to host multiple tenants. Members of the same tenant should be able to
+only see each other when trying to share resources. A user can only be a member
+of a single tenant. Moving a user from one tenant to another is not supported.
+For the scope of this ADR we assume the tenants are overall rather small
+(sometimes just a single user, often less than 10). Also there is no need to
+support per tenant groups of users for now.
```

Comment:
I don't know if disabling group management is currently just a config flag, or if disabling this requires more work than adding tenants to groups.

Reply:
Since all users are centrally managed. There should not be any user that has the privileges to edit users and groups. Apart maybe from assigning roles.

Actually there are a few details we still need to sort out for that. How are roles being assigned? Since the IDP is centrally managed it might not be possible to configure the roles via OIDC claims. That means we would at least need to have one user per tenant that is able change the roles of other users in the tenant. E.g. to have SpaceAdmins.